python/ray/_private/services.py

```diff
@@ -834,8 +834,8 @@ def start_ray_process(
     use_valgrind_profiler: bool = False,
     use_perftools_profiler: bool = False,
     use_tmux: bool = False,
-    stdout_file: Optional[str] = None,
-    stderr_file: Optional[str] = None,
+    stdout_file: Optional[IO[AnyStr]] = None,
+    stderr_file: Optional[IO[AnyStr]] = None,
```

Comment:
Are we IO[AnyStr] or IO[str] ?

Reply:
I'm copying the type annotation here: https://github.com/ray-project/ray/blob/3496c5b4b1176919722501ab1b39ce42dcc31638/python/ray/_private/node.py#L851
which is the return type where we get file handle, would prefer to keep them consistent.